sdk/monitor/azure-opentelemetry-exporter-azuremonitor/azure/opentelemetry/exporter/azuremonitor/export/metrics/_exporter.py

```diff
@@ -0,0 +1,181 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+import logging
+from typing import Sequence
+
+from opentelemetry.sdk.metrics.export import (
+    ExportRecord,
+    MetricsExporter,
+    MetricsExportResult,
+)
+from opentelemetry.sdk.metrics.export.aggregate import (
+    HistogramAggregator,
+    MinMaxSumCountAggregator,
+    ValueObserverAggregator,
+)
+from opentelemetry.sdk.util import ns_to_iso_str
+
+from azure.opentelemetry.exporter.azuremonitor import _utils
+from azure.opentelemetry.exporter.azuremonitor._generated.models import (
+    MetricDataPoint,
+    MetricsData,
+    MonitorBase,
+    TelemetryItem
+)
+from azure.opentelemetry.exporter.azuremonitor.export._base import (
+    BaseExporter,
+    ExportResult,
+    get_metrics_export_result,
+)
+
+logger = logging.getLogger(__name__)
+
+__all__ = ["AzureMonitorMetricsExporter"]
+
+
+class AzureMonitorMetricsExporter(BaseExporter, MetricsExporter):
+    """Azure Monitor metrics exporter for OpenTelemetry.
+
+    :param options: Exporter configuration options.
+    :type options: ~azure.opentelemetry.exporter.azuremonitor.options.ExporterOptions
+    """
+
+    def export(
+        self, export_records: Sequence[ExportRecord]
+    ) -> MetricsExportResult:
```

Comment:
add **kwargs?

Reply:
`export` is never called by the user so I do not believe there is a need for `**kwargs`